docs/examples/collapse/custom-icon.vue

```diff
@@ -57,9 +57,9 @@
 <script lang="ts" setup>
 import { ref } from 'vue'
 import { CaretRight } from '@element-plus/icons-vue'
-
+import type { CollapseModelValue } from 'element-plus'
 const activeNames = ref(['1'])
```

Comment:
Maybe should add an escape line here
```suggestion
import type { CollapseModelValue } from 'element-plus'

const activeNames = ref(['1'])
```